.chloggen/tempo-query_ReadinessProbe.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add ReadinessProbe to tempo-query
```

Comment:
Why is this being added? What are the consequences of this patch?

Reply:
Instead of showing that the pod is ready and handling the failure internally, the k8s api will show that its not ready. K8s will do regular calls to the gRPC endpoint to verify that its alive.

Reply:
Yes, and we should document https://github.com/grafana/tempo-operator/pull/1061#issuecomment-2413275405 in the changelog. Impact of adding readiness probe to our deployment. 

I would also say that this is more a bug_fix than enhancement.

Reply:
e.g. improves reliability at startup, avoiding lost data